src/main/java/com/google/codeu/data/Message.java

```diff
@@ -56,4 +61,7 @@ public String getText() {
   public long getTimestamp() {
     return timestamp;
   }
+
+  //add getter for recipient that provides access to the field
+  public String getRecipient(){ return recipient; }
```

Comment:
Looks like another thing that should be caught by reformatting the file this is in.

Reply:
I believe this is fixed.